### ISSUE TITLE
x/build: two darwin builders are missing

### ISSUE THREAD

**Issue by dmitshur** [golang/go#39378]
From https://farmer.golang.org/#health:

```
# "macs" status: MacStadium Mac VMs
# Notes: https://github.com/golang/build/tree/master/env/darwin/macstadium
Warn: macstadium_host04a missing, not seen for 14h47m29s
Warn: macstadium_host04b missing, not seen for 14h46m54s
Warn: 2 machines missing, 10% of capacity
Warn: makemac daemon: vm.destroy("mac_10_14_host04a") = govc vm.destroy ...: exit status 1, govc: Unable to communicate with the remote host, since it is disconnected.

Warn: makemac daemon: vm.destroy("mac_10_12_host04b") = govc vm.destroy ...: exit status 1, govc: Unable to communicate with the remote host, since it is disconnected.
```

Possibly similar or related to #39306.

/cc @toothrot @cagedmantis @andybons

**Comment by cagedmantis** [comment]
This has been happening often lately. It might be worth taking a deeper look if it happens again.

**Comment by cagedmantis** [comment]
This has been resolved by hard booting host #4.
